python/cudf/cudf/tests/test_parquet.py

```diff
@@ -281,13 +282,16 @@ def num_row_groups(rows, group_size):
 
 
 @pytest.mark.parametrize("row_group_size", [1, 5, 100])
-def test_parquet_read_row_group(tmpdir, pdf, row_group_size):
+def test_parquet_read_row_group_concat(tmpdir, pdf, row_group_size):
     fname = tmpdir.join("row_group.parquet")
     pdf.to_parquet(fname, compression="gzip", row_group_size=row_group_size)
 
     num_rows, row_groups, col_names = cudf.io.read_parquet_metadata(fname)
 
-    gdf = [cudf.read_parquet(fname, row_group=i) for i in range(row_groups)]
+    gdf = [
+        cudf.read_parquet([fname, fname], row_group_list=[[i], []])
```

Comment:
If it's not too much of a pain, it would be nice to test the new `row_group_list` mechanism for a single file, and for non-contiguous lists of row-group indices.  For example, in the near future, `dask.dataframe`/`dask_cudf` will likely want to pass in something like `row_group_list=[0,1,3,8,9]` for a single file.

Reply:
added a test that alternates between the two sources. Modified the basic row group list test back to a single source, it did not increase code coverage with the newly added cases.